oci/spec_test.go

```diff
@@ -63,7 +63,7 @@ func TestGenerateSpec(t *testing.T) {
 		}
 	} else {
 		if s.Windows == nil {
-			t.Fatal("Windows section of spec not filled on on Windows platform")
+			t.Fatal("Windows section of spec not filled on Windows platform")
```

Comment:
the typo here is that the first "on" should have been "**in**", so that it says "Windows section of spec not filled **in** on Windows platform"; please update.